File: my-app/src/app/common/matcheventrequest.ts
export class Matcheventrequest {
  constructor(public id:number,public minute:number,public type:string,public club_id:number,public player_id:number,public assist_id:number,public subsitutePlayer_id:number,public isPenalty:boolean,public isOwnGoal:boolean,public isRedCard:boolean,public isYellowCard:boolean,public description:string,public match_id:number ) {
  }
}

File: FootballApp/src/main/java/com/football/dev/footballapp/dto/MatchEventRequest.java
package com.football.dev.footballapp.dto;

import com.football.dev.footballapp.models.enums.TypeStatus;

public record MatchEventRequest(Long id ,
                                Integer minute,
                                TypeStatus type,
                                Long club_id,
                                Long player_id,
                                Long assist_id,
                                Long subsitutePlayer_id,
                                boolean isPenalty,
                                boolean isOwnGoal,
                                boolean isRedCard,
                                boolean isYellowCard,
                                String description,
                                Long match_id) {

}
